File: C/19.cpp
#include <stdio.h>
#include <conio.h>
#include <stdlib.h>
#include <locale.h>

int main (void) {
	system ("cls");
	setlocale (LC_ALL, "Portuguese");
	int massaPrimeiro,massaSegundo,massaTerceiro,protonUm,protonDois,protonTres,neutronUm,neutronDois,neutronTres; 
	printf ("\n\nEste programa lê as caracteristicas de três elementos químicos e os determina!");
	
 	printf ("\n\nDigite a massa do do primeiro elemento: ");
	scanf ("%d", &massaPrimeiro);
 	printf ("\n\nDigite a massa do do segundo elemento: ");
	scanf ("%d", &massaSegundo);
 	printf ("\n\nDigite a massa do do terceiro elemento: ");
	scanf ("%d", &massaTerceiro);

 	printf ("\n\nDigite a quantidade de prótons do primeiro elemento: ");
	scanf ("%d", &protonUm);
 	printf ("\n\nDigite a quantidade de prótons do segundo elemento: ");
	scanf ("%d", &protonDois);
 	printf ("\n\nDigite a quantidade de prótons do terceiro elemento: ");
	scanf ("%d", &protonTres);	

 	printf ("\n\nDigite a quantidade de nêutron do terceiro elemento: ");
	scanf ("%d", &neutronUm);	
 	printf ("\n\nDigite a quantidade de nêutron do terceiro elemento: ");
	scanf ("%d", &neutronDois);	
 	printf ("\n\nDigite a quantidade de nêutron do terceiro elemento: ");
	scanf ("%d", &neutronTres);	
	system("cls");
	if(massaPrimeiro==massaSegundo){
		printf("Os elementos 1 e 2 são isobaros!");
	}
	if(massaPrimeiro==massaTerceiro){
		printf("Os elementos 1 e 3 são isobaros!");
	}
	if(massaSegundo==massaTerceiro){
		printf("Os elementos 2 e 3 são isobaros!");
	}
	
	
	if(protonUm==neutronDois){
		printf("Os elementos 1 e 2 são isotonos!");
	}
	if(protonUm==neutronTres){
		printf("Os elementos 1 e 3 são isotonos!");
	}
	if(neutronDois==neutronTres){
		printf("Os elementos 2 e 3 são isotonos!");
	}
	
		
	if(massaPrimeiro==protonDois){
		printf("Os elementos 1 e 2 são isotonos!");
	}
	if(massaPrimeiro==protonTres){
		printf("Os elementos 1 e 3 são isotonos!");
	}
	if(protonDois==protonTres){
		printf("Os elementos 2 e 3 são isotonos!");
	}
	printf ("\n\nObrigado por utilizar nosso sistema!\n\n");
	system ("Pause");
	return 0;
}

File: Python/19.py
printf = print
printf("\n\nEste programa lê as caracteristicas de três elementos químicos e os determina!")

massaPrimeiro = int(input("\n\nDigite a massa do do primeiro elemento: "))

massaSegundo = int(input("\n\nDigite a massa do do segundo elemento: "))

massaTerceiro = int(input("\n\nDigite a massa do do terceiro elemento: "))

protonUm = int(input("\n\nDigite a quantidade de prótons do primeiro elemento: "))

protonDois = int(input("\n\nDigite a quantidade de prótons do segundo elemento: "))

protonTres = int(input("\n\nDigite a quantidade de prótons do terceiro elemento: "))

neutronUm = int(input("\n\nDigite a quantidade de nêutron do terceiro elemento: "))

neutronDois = int(input("\n\nDigite a quantidade de nêutron do terceiro elemento: "))

neutronTres = int(input("\n\nDigite a quantidade de nêutron do terceiro elemento: "))

if (massaPrimeiro == massaSegundo):
    printf("Os elementos 1 e 2 são isobaros!")

if (massaPrimeiro == massaTerceiro):
    printf("Os elementos 1 e 3 são isobaros!")

if (massaSegundo == massaTerceiro):
    printf("Os elementos 2 e 3 são isobaros!")

if (protonUm == neutronDois):
    printf("Os elementos 1 e 2 são isotonos!")

if (protonUm == neutronTres):
    printf("Os elementos 1 e 3 são isotonos!")

if (neutronDois == neutronTres):
    printf("Os elementos 2 e 3 são isotonos!")

if (massaPrimeiro == protonDois):
    printf("Os elementos 1 e 2 são isotonos!")

if (massaPrimeiro == protonTres):
    printf("Os elementos 1 e 3 são isotonos!")

if (protonDois == protonTres):
    printf("Os elementos 2 e 3 são isotonos!")

printf("\n\nObrigado por utilizar nosso sistema!\n\n")
